askama_shared/src/generator.rs

```diff
@@ -1513,19 +1492,35 @@ impl<'a, S: std::hash::BuildHasher> Generator<'a, S> {
         DisplayWrap::Unwrapped
     }
 
-    fn visit_target(&mut self, buf: &mut Buffer, target: &'a Target) {
-        match *target {
+    fn visit_target(&mut self, buf: &mut Buffer, var: &Target<'a>) {
+        match *var {
```

Comment:
```suggestion
        match var {
```

Pretty sure you can remove the `*`, it's not needed anymore :)